charts/scalardb-cluster/templates/scalardb-cluster/deployment.yaml

```diff
@@ -59,13 +59,19 @@ spec:
           {{- end }}
           resources:
             {{- toYaml .Values.scalardbCluster.resources | nindent 12 }}
+          startupProbe:
+            exec:
+              command:
+                - /usr/local/bin/grpc_health_probe
+                - -addr=:60053
+            failureThreshold: 60
+            periodSeconds: 5
```

Comment:
I set the timeout value to 300sec (`failureThreshold * periodSeconds = 300sec`). If the pod won't be `Ready` state after 300sec, Kubernetes terminates the pod.